core/client/src/test/java/alluxio/client/file/FileOutStreamTest.java

```diff
@@ -97,11 +104,13 @@
   @Before
   public void before() throws Exception {
     ClientTestUtils.setSmallBufferSizes();
+    mDelegationEnabled = Configuration.getBoolean(PropertyKey.USER_UFS_DELEGATION_ENABLED);
```

Comment:
I would prefer to set this with a `ConfigurationRule`. I don't think it makes sense to support both delegation enabled and disabled because we will only actually test the default one.